### PR TITLE
fix(install): remove escapes on heading when coloring is not supported

### PR DESCRIPTION
**Pull request by paulozulato**
<!-- Thank you for sending a pull request :heart: -->

## Current behavior

Escape characters are still being printed on heading even when terminal doesn't support coloring, resulting in strange characters being printed:
```
\033[0;33m   GET STARTED:\033[0m                                                                                                                                                                                                                            
   Ockam Command is installed at: "$HOME/.ockam/bin/ockam"     
```

## Proposed changes

Removed escape characters in case that colorizing is not supported:
```
   GET STARTED:
   Ockam Command is installed at: "$HOME/.ockam/bin/ockam"
```

## Checks

<!-- To help us review and merge this pull request quickly, please confirm the following:  -->

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. The Ockam repo maintains a linear commit history so we merge PRs by rebasing them to the develop branch. Rebasing to the latest develop branch and force pushing to your PR branch is okay.
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

<!-- Looking forward to merging your contribution!! -->

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ heading() {
     # shellcheck disable=SC2059
     printf "${_orange}   $1${_no_color}\n"
   else
-    echo "${_orange}   $1${_no_color}"
+    echo "   $1"
```
Comment (metaclips) [inline install.sh:88]: printf should handle this correctly
```bash
printf "${_orange} $1${_no_color}"
```

**Review by metaclips — DISMISSED** [review]
Thanks for an amazing PR

**Review by mrinalwadhwa — APPROVED** [review]

**Comment by metaclips** [comment]
There's a PR that adds same fix https://github.com/build-trust/ockam/pull/5966, I made both PRs co-authors.

**Comment by mrinalwadhwa** [comment]
@paulozulato we landed the change as part of https://github.com/build-trust/ockam/pull/5966, congratulations on an awesome first contribution.

**Comment by paulozulato** [comment]
thank you very much, @mrinalwadhwa

**Pull request closed without merging**